space-provider/space-framework.md

```diff
@@ -0,0 +1,117 @@
+This document includes the text from the space-framework design document
+
+The full design document can be found here: https://docs.google.com/document/d/1WZCxlGoaCZVymqMhLCa5JZeURdCTzju8C5ABS8V4lW0/edit?usp=sharing
+
+
+# Space support for KubeStellar
+
+## Background
+
+KubeStellar architecture leverages spaces as the container building block for various purposes. For example, the separation between the KubeStellar service provider and the inventory is done by using different spaces for the inventory and the service provider. 
+
+The KubeStellar design currently 
+
+1. Separate different user workloads into different spaces (workload management spaces)
+2. Synchronise each EdgeCluster into a dedicated space (called the mailbox space ). 
+
+In the future some of the above design may change (e.g., the need for  1:1 mapping  of MBLC:EdgeCluster or the need for separate WMLC for each workload), however we believe some usage of spaces will still be needed.
+
+Today KubeStellar is using KCP workspaces as spaces and is practically built on top of KCP. 
+
+This proposal tries to disconnect KubeStellar architecture and internal design and the underlying support for spaces. 
```

Comment:
Everything is connected to everything, more or less directly.  I find talking about "disconnect" here to be confusing.  I am more comfortable with words about adding an intermediary that allows the multi-cluster configuration management code to consume space functionality from a variety of providers of that functionality.

Remember also the issue of the scope of the name "KubeStellar".  As long as it is a large scope that encompasses the space framework then it is not a narrow thing refers only to the multi-cluster configuration management.